src/main/java/com/okta/developer/domain/User.java

```diff
@@ -83,9 +83,8 @@ public String getLogin() {
         return login;
     }
 
-    // Lowercase the login before saving it in database
     public void setLogin(String login) {
-        this.login = StringUtils.lowerCase(login, Locale.ENGLISH);
```

Comment:
I think this is what was blocking the `sub` claim looking from working when the user logs in (and I was too tired to notice it yesterday, and hacked up that email work around)